src/common/prometheus_client.cc

```diff
@@ -32,4 +32,19 @@ DEFINE_PROMETHEUS_HISTOGRAM(knowhere_range_search_latency, "range search latency
 DEFINE_PROMETHEUS_HISTOGRAM(knowhere_ann_iterator_init_latency, "ann iterator init latency in knowhere (ms)")
 DEFINE_PROMETHEUS_HISTOGRAM(knowhere_search_topk, "knowhere search topk")
 
+const prometheus::Histogram::BucketBoundaries hnswBitsetRatioBuckets = {0.05, 0.10, 0.15, 0.20, 0.25, 0.30, 0.35,
+                                                                        0.40, 0.45, 0.50, 0.55, 0.60, 0.65, 0.70,
+                                                                        0.75, 0.80, 0.85, 0.90, 0.95, 1.00};
+DEFINE_PROMETHEUS_HISTOGRAM_WITH_BUCKETS(knowhere_hnsw_bitset_ratio,
+                                         "knowhere HNSW bitset ratio for search and range search",
+                                         hnswBitsetRatioBuckets)
+
+const prometheus::Histogram::BucketBoundaries hnswSearchHopBuckets = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 100};
```

Comment:
this might be too small for large segments, and the interval is too small as well.

Is there any limitations on the number of buckets?